js-exercises/balanced-braces/balancedBraces.js

```diff
@@ -0,0 +1,34 @@
+function balancedBraces (str) {
+  let stack = [];
+
+    let openBracesArray = {
```

Comment:
The wording is wrong. The value is an object and not an array.

Reply:
Fixed this comment and committed code.